### PR TITLE
Fixed all Level 1 and Level 2 bugs assigned to me, except renaming all the labs

### PR DESCRIPTION
**Pull request by markkramerus**

### CODE REVIEW

File: spec/oncocore.txt [hunk 1]
```diff
@@ -134,8 +134,8 @@ EntryElement:   Tumor
 Based on:		BodyStructure
 Concept:        SCT#395557000 "Tumor finding (finding)"
 Description:    "The presence of an abnormal mass of tissue (neoplasm) that results when cells divide more than they should or do not die when they should. Tumors may be benign (not cancer), or malignant (cancer). (source: NCI Dictionary)."
-                Code from HistologyMorphologyBehaviorVS
-0..1			RelatedCancerCondition
+                Code from HistologyMorphologyBehaviorVS if covered
+0..1			ref(PrimaryCancerCondition)
```
Comment (rutemartins) [inline spec/oncocore.txt]: A `Tumor` can also be associated with a `SecondaryCancerCondition`, so we should allow for that reference here as well.

File: spec/oncocore_icd10_vs.txt [hunk 2]
```diff
@@ -2549,8 +2550,11 @@ ICD10CM#D4989       "Neoplasm of unspecified behavior of other specified sites"
...
+ValueSet:   SecondaryCancerDisorderVS
+Description:  "Types of secondary malignant neoplastic disease, coded in SNOMED CT, ICD-10-CM, or ICD-O-3. 
+* SNOMED CT coding: Use a code from the disorder hierarchy under secondary malignant neoplastic disease (SNOMED-CT 128462008). 
+* ICD-10-CM coding: Use one of the codes given in this value set representing secondary malignant neoplasms and neoplasms of uncertain or unspecified behavior. If body site is not precoordinated (implied by the code), it should be specified separately using the body location.
```
Comment (rutemartins) [inline spec/oncocore_icd10_vs.txt]: This value set should not include neoplasms of uncertain or unspecified behavior. These are assumed to be primary in ICD-10-CM. If you don't know whether it's malignant, you certainly can't tell if it's secondary.

File: spec/oncocore_vs.txt [hunk 3]
```diff
@@ -1,21 +1,16 @@
...
 CodeSystem:	    MTH = http://ncimeta.nci.nih.gov
 CodeSystem:     ICD10CM = http://hl7.org/fhir/sid/icd-10-cm
-CodeSystem:     ICDO3MORPH = http://codes.iarc.fr/codegroup/2
-CodeSystem:     ICDO3TOPO = http://codes.iarc.fr/topography 
-
 
 ValueSet:     CancerBodyLocationVS
-Description:  "Codes describing the location of primary or secondary cancer. The value set includes all codes from ICD-O-3 topology and SNOMED-CT body locations."
-Includes codes from ICDO3TOPO
+Description:  "Codes describing the location of primary or secondary cancer. The value set includes all codes SNOMED-CT body structure hierarchy. The cancer body location code can also come from ICD-O-3 topography codes, however, those codes are not included here due to intellectual property restrictions. Because the value set binding is extensible in the information model, use of ICD-O-3 topography codes conform to mCODE requirements."
```
Comment (rutemartins) [inline spec/oncocore_vs.txt]: Would not include notes on information model in value set description.

File: spec/oncocore_vs.txt [hunk 4]
```diff
@@ -1,21 +1,16 @@
...
 Includes codes descending from SCT#123037004  "Body Structure"
 
 ValueSet:   HistologyMorphologyBehaviorVS
-Description: "The histologic type codes can be taken either from ICD-O-3 morphology codes (including behavior suffix) or SNOMED CT neoplasm morphology codes. For primary cancers, the ICD-O-3 behavior suffix should be /1, /2, or /3. For secondary cancers, the ICD-O-3 code should be /6 or /9."
-Includes codes from ICDO3MORPH
+Description: "The histologic type codes can be taken either from ICD-O-3 morphology codes (including behavior suffix) or SNOMED CT neoplasm morphology codes. For primary cancers, the ICD-O-3 behavior suffix should be /1, /2, or /3. For secondary cancers, the ICD-O-3 behavior suffix should be /6 or /9. The ICD-O-3 codes are not included here due to intellectual property restrictions, but the value set binding is extensible, so ICD-O-3 codes conform to mCODE requirements."
```
Comment (rutemartins) [inline spec/oncocore_vs.txt]: Would not include notes on information model in value set description.

File: spec/oncocore_icd10_vs.txt [hunk 5]
```diff
@@ -1,15 +1,13 @@
...
-Namespace:	oncocore
+Namespace:	onco.core
 
 CodeSystem:     SCT = http://snomed.info/sct
 CodeSystem:     ICD10CM = http://hl7.org/fhir/sid/icd-10-cm
-CodeSystem:     ICDO3MORPH = http://codes.iarc.fr/codegroup/2
 
 ValueSet:      CancerDisorderVS
-Description:   "A broad cancer-related value set containing both primary and secondary tumor types, with codes from ICD-10, ICD-O-3, and SNOMED CT, including both diagnosis and histology/morphology/behavior codes."
+Description:   "A broad cancer-related value set containing both primary and secondary tumor types, with codes from ICD-10 and SNOMED CT, including both diagnosis and histology/morphology/behavior codes. ICD-O-3 codes are not included here due to intellectual property restrictions, but the value set binding is extensible, so ICD-O-3 codes conform to mCODE requirements."
```
Comment (mlterryMitre) [inline spec/oncocore_icd10_vs.txt]: change from `ICD-10` to `ICD-10-CM`.

File: spec/oncocore_vs.txt (hunk 4 above)
Comment (mlterryMitre) [reply spec/oncocore_vs.txt]: I actually read this as a pre-coordinated term in the value set so I would keep this description here as-is.

**Pull request merged**